Update preset-security-policies.md (#2440)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/office-365-security/preset-security-policies.md

1. **Strict protection** preset security policy
2. **Standard protection** preset security policy
3. Custom policies

> 4. Default Policy

Reviewer: ```suggestion
4. Default policy
```

> In other words, the settings of the **Strict protection** policy override the settings of the **Standard protection** policy, which overrides the settings from any other related policies.

Reviewer: ```suggestion
In other words, the settings of the **Strict protection** policy override the settings of the **Standard protection** policy, which overrides the settings from custom policies and the default policy.
```
